### DISCUSSION TITLE
Schema updates?

### DISCUSSION THREAD

**Discussion by Brinkley100** [CMSgov/price-transparency-guide#172] [Q&A] [answered]
We heard that CMS would be finalizing the JSON schema by the end of this week and would be providing clarifications to some of the open issues/questions. Did anyone of you also get this information? It is almost the end of this week and there seems to be no movement on that front. So, just wondering?

**Comment by Ginger228** [comment]
> We heard that CMS would be finalizing the JSON schema by the end of this week and would be providing clarifications to some of the open issues/questions. Did anyone of you also get this information? It is almost the end of this week and there seems to be no movement on that front. So, just wondering?

Thanks for bringing this up.  I also heard CMS would be updating GitHub with updates to the schema and the compliance date.

**Reply by ericsilvertx** [reply]
Looking forward to the update on the schema!

**Reply by ericsilvertx** [reply]
We did not see any schema updated and it is monday

**Comment by shaselton-usds** [comment, marked as answer]
Closing this thread.

Version 1.0 was finalized on March 1st, 2022. Updates will continue to be made to the schema as they are needed, but compliance will be based on version 1.0.
